packages/core/src/payment/strategies/apple-pay/apple-pay-payment.mock.ts

```diff
@@ -33,5 +34,5 @@ export class MockApplePaySession {
         return event;
     }
 
-    oncancel() { }
+    oncancel() { noop }
```

Comment:
Not sure if this is correct.

Reply:
Changed it to do a return instead `oncancel() { return }`